CONTRIBUTING.md

```diff
@@ -0,0 +1,31 @@
+# How to contribute
+
+We'd love to accept your patches and contributions to this project. There are
+just a few small guidelines you need to follow.
+
+## Contributor License Agreement
+
+Contributions to this project must be accompanied by a Contributor License
+Agreement. You (or your employer) retain the copyright to your contribution,
+this simply gives us permission to use and redistribute your contributions as
+part of the project. Head over to <https://cla.developers.google.com/> to see
```

Comment:
Where did you find this text? Google generally tries not to associate itself with projects unless they need to.

Reply:
Found this in other Google Open Source projects. This one specifically came from https://github.com/google/earthenterprise/blob/master/CONTRIBUTING.md

Reply:
Hmm that looks like an "official" Google open-source project, I don't think this project would require the CLA and there might be some objections to the implication of asking contributors to sign it.